### PR TITLE
Add DZ60 ANSI layout with arrows and F keys

### PR DESCRIPTION
**Pull request by BenRoe**
I testet the keymap on my dz60 keyboard.

### CODE REVIEW

File: keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#include "dz60.h"
```
Comment (drashna) [inline keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c]: Could you change this to `#include QMK_KEYBOARD_H`?

File: keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   * |-----------------------------------------------------------------------------------------+
+   * | Caps    |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |    Enter    |
+   * |-----------------------------------------------------------------------------------------+
+   * | Shift     |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  | RSh |  U  | DEL |
+   * |-----------------------------------------------------------------------------------------+
+   * | Ctrl |  Cmd  |  Alt  |              Space                | Cmd | RAlt |  L  |  D  |  R  |
+   * `-----------------------------------------------------------------------------------------'
+   */
+
+  KEYMAP_2_SHIFTS(
```
Comment (drashna) [inline keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c]: Could you change the `KEYMAP_2_shifts` to `LAYOUT_2_shifts`?

File: keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+   * |-----------------------------------------------------------------------------------------+
+   * |         | BL T| BL M| BL+ | BL- |     |     |     |     |     |     |     |             |
+   * |-----------------------------------------------------------------------------------------+
+   * |           |     |     |     |     |     |     |     |     |     |     |     |     |     |
+   * |-----------------------------------------------------------------------------------------+
+   * |      |       |       |                                   |     |      |     |     |     |
+   * `-----------------------------------------------------------------------------------------'
+   */
+
+  KEYMAP_DIRECTIONAL(
```
Comment (drashna) [inline keyboards/dz60/keymaps/ansi_arrow_fkeys/keymap.c]: Could you change the `KEYMAP_2_shifts` to `LAYOUT_2_shifts`?

**Comment by BenRoe** [comment]
Hi,
i change the code to the new layout syntax and added added a readme.
Also changed the folder name.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**